src/analysisd/decoders/security_configuration_assessment.c

```diff
@@ -1356,14 +1358,16 @@ static int CheckEventJSON(cJSON *event, cJSON **scan_id, cJSON **id, cJSON **nam
 
     } else {
 
-        if( *id = cJSON_GetObjectItem(*check, "id"), !*id) {
+        if ((*id = cJSON_GetObjectItem(*check, "id")) == NULL)
```

Comment:
Please, let's keep the typical format we use in the rest of the functions, with the bracket on the same line as the header.

```suggestion
        if ((*id = cJSON_GetObjectItem(*check, "id")) == NULL) {
```